provisioning/transport/mqtt/src/ProvisioningChannelHandlerAdapter.cs

```diff
@@ -309,7 +322,25 @@ private async Task VerifyPublishPacketTopicAsync(IChannelHandlerContext context,
             {
                 if (Enum.TryParse(match.Groups[1].Value, out HttpStatusCode statusCode))
                 {
-                    if (statusCode >= HttpStatusCode.BadRequest)
+                    if (statusCode == HttpStatusCode.BadRequest)
```

Comment:
>BadRequest [](start = 53, length = 10)

even this condition appears to be covered by line 347

Reply:
I'll revert this to maintain the current error parsing then